### DISCUSSION TITLE
Emissa and epubgen2

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2635] [Developers corner]
Does anyone know why Emissa.pl is in cgi-bin/missa instead of standalone/tools/epubgen2 where it seems obvious it belongs?

Does anyone know if epubgen/epubgen.sh is still relevant? Or does epubgen2 supersede it?

**Comment by gregordick** [comment]
That file was introduced in
```
commit 7c7b4085781c6f5e1e9dd9f572e62c45dae12a03
Author: Geremia <Geremia@users.noreply.github.com>
Date:   Sun Jun 14 16:24:11 2020 -0700

    -m option for epubgen2.sh to include Mass propers
```
@Geremia: Do you have any insights?
